docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,272 @@
+# Installing Canonical Kubernetes in airgapped environments
+
+There are situations where it is necessary or desirable to run Canonical 
+Kubernetes on a machine that is not connected to the internet. 
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide documents any necessary extra preparation for airgap deployments,
+as well the steps that are needed to successfully deploy Canonical Kubernetes
+in such environments.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta --basename k8s
+sudo snap download core20 --basename core20
+```
+
+The [core20][Core20] and `k8s` snap are downloaded. `core20.assert` and 
+`k8s.assert` are necessary to verify the integrity of the snap packages.
```

Comment:
Besides the snaps, this will also download the corresponding `assert` files which are necessary to verify the integrity of the packages.